pkg/virt-launcher/virtwrap/network/dhcpv6/dhcpv6.go

```diff
@@ -112,7 +112,12 @@ func (h *DHCPv6Handler) ServeDHCPv6(conn net.PacketConn, peer net.Addr, m dhcpv6
 	switch msg.Type() {
 	case dhcpv6.MessageTypeSolicit:
 		log.Log.V(4).Info("DHCPv6 - the request has message type Solicit")
-		response, err = dhcpv6.NewAdvertiseFromSolicit(msg, h.modifiers...)
+		if msg.GetOneOption(dhcpv6.OptionRapidCommit) == nil {
+			response, err = dhcpv6.NewAdvertiseFromSolicit(msg, h.modifiers...)
+		} else {
```

Comment:
Could you please explain what does it mean in the commit message? (reply to dhcp solict with rapid commit)

Reply:
done.